src/formats.rs

```diff
@@ -535,6 +535,13 @@ formats! {
     extension = "dex"
     kind = Executable
 
+    format = DashPlaylist
+    name = "MPEG-DASH manifest"
+    short_name= "DASH"
+    media_type = "application/dash+xml"
+    extension = "dash"
```

Comment:
I think that the "mpd" extension would be better in this context, as indicated by IANA in its format specification.